seed/views/main.py

```diff
@@ -2017,7 +2020,7 @@ def progress(request):
     progress_key = json.loads(request.body).get('progress_key')
 
     if cache.get(progress_key):
-        result = cache.get(progress_key)
+        result = get_cache(progress_key)
```

Comment:
Is this correct? Aren't sin't `cache.get` the new method for getting the cache data?